board.js

```diff
@@ -0,0 +1,64 @@
+class	Board
+{
+	constructor()
+	{
+		this.borderStyle = "1px solid";
+		this.backgroundColor = "#FFF";
+		this.irlWidth = 274;
+		this.irlHeight = 152.5;
+		this.irlRatio = this.irlHeight / this.irlWidth;
+		this.canvas = document.getElementById("board");
+		this.ctx = this.canvas.getContext("2d");
+		this.ctx.font = "30px Arial";
+		this.ctx.fillStyle = "#000";
+		this.ctx.textAlign = "center";
+		this.dim = new Dimension();
+		this.game = undefined;
+		this.setHeight = () =>
+		{
+			this.dim.height = this.irlRatio * this.dim.width;
+			return (this.dim.height);
+		};
+		this.updateSizeAfterResize = () =>
+		{
+			let prevWidth = this.dim.width; // this.width 
+			let	prevHeight = this.dim.height; // this.height obj Borad 
```

Comment:
indentation Failed, when react-ts project will be implemented, we can see directly the good practice learned from 42 with the linter